packages/terra-table/src/TableSubheader.jsx

```diff
@@ -19,21 +22,14 @@ const TableSubheader = ({
   content,
   colSpan,
   ...customProps
-}) => {
-  const contentClassName = classNames([
-    'terra-Table-subheader',
-    customProps.className,
-  ]);
-
+}) => (
   // count is based on the number of columns and assigned in the table component which contains this subheader
-  return (
-    <tr className="terra-Table-subheaderRow">
-      <td {...customProps} className={contentClassName} colSpan={colSpan}>
-        {content}
-      </td>
-    </tr>
-  );
-};
+  <tr data-class="subheader-row">
+    <td {...customProps} className={cx('subheader', customProps.className)} colSpan={colSpan}>
```

Comment:
I just will use our locally scoped styling. I checked with code base. It appends fill class twice, one if from `fill={true}` and one is from `class="fill"`.
<img width="911" alt="screen shot 2017-07-26 at 10 23 23 am" src="https://user-images.githubusercontent.com/3724179/28629248-89fcd94e-71ec-11e7-95ea-a3987e9116b4.png">